instance/migrations/0114_introduce_base_playbook_name.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2019-02-27 15:34
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models, transaction
+from django.db.models import F
```

Comment:
@giovannicimolin, this is an unused import

Reply:
Done

Reply:
Fixed.